### PR TITLE
Update with-office-365.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/with-office-365.md [hunk 1]
```diff
@@ -186,9 +186,11 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
 > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
 
    ``` Powershell
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-office-365.md:188]:
```suggestion
When using the Teams PowerShell module:
   ``` Powershell
```

File: Teams/rooms/with-office-365.md [hunk 2]
```diff
@@ -186,9 +186,11 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
...
-   Import-Module -Name MicrosoftTeams  
-   $cssess = New-CsOnlineSession -Credential $cred  
-   Import-PSSession $cssess -AllowClobber
+   # When using Teams PowerShell Module
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-office-365.md:189]:
```suggestion
```

File: Teams/rooms/with-office-365.md [hunk 3]
```diff
@@ -186,9 +186,11 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
...
+
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-office-365.md:190]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**